auto-generated/vector-crypto/intrinsic_funcs/00_zvbb_-_vector_bit-manipulation_used_in_cryptography.md

```diff
@@ -0,0 +1,581 @@
+
+## Zvbb - Vector Bit-manipulation used in Cryptography:
```

Comment:
FYI, I just opened https://github.com/riscv/riscv-crypto/issues/342 to get this name changed to not mention cryptography.

Reply:
Name has been changed in the spec and also a subset `Zvkb` has been split out of `Zvbb` and could be documented separately.

Reply:
```suggestion
## Zvbb - Vector Basic Bit-manipulation used in Cryptography:
```

Reply:
Shouldn't the "used in Cryptography" be dropped?

Reply:
Note that some of the instructions below are indeed defined in `Zvkb` (Vector Bit-manipulation used in Cryptography)

Reply:
Should the file be split into Zvbb and Zvkb?

Reply:
The thing is Zvkb is a subset of Zvbb so the `Zvbb` file will not contain all Zvbb instructions. So I am not sure it makes sense to split the file.